src/main/java/io/confluent/connect/hdfs/avro/AvroRecordWriterProvider.java

```diff
@@ -86,6 +88,14 @@ public void write(SinkRecord record) {
           }
         } catch (IOException e) {
           throw new DataException(e);
+        } catch (AvroRuntimeException ex) {
+          // Making schema null to invalidate file corrupted due to connection failure
+          schema = null;
+          /*
+          Wrapped into ConnectException so that setResetTimeout() is called in the
```

Comment:
Is it setRetryTimeout or setResetTimeout?

Reply:
Its setRetryTimeout(). changed it